src/Components/ImagesTable/ImageLink.js

```diff
@@ -1,126 +1,45 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import PropTypes from 'prop-types';
-
-import {
-  Button,
-  Popover,
-  Text,
-  TextContent,
-  TextVariants,
-} from '@patternfly/react-core';
-import { DownloadIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core';
+import { useLoadModule } from '@scalprum/react-core';
+import ImageLinkDirect from './ImageLinkDirect';
 
 const ImageLink = (props) => {
-  const fileExtensions = {
-    vsphere: '.vmdk',
-    'guest-image': '.qcow2',
-    'image-installer': '.iso',
-  };
+  const [wizardOpen, openWizard] = React.useState(false);
+  const [{ default: ProvisioningWizard }, error] = useLoadModule(
+    {
+      appName: 'provisioning', // optional
+      scope: 'provisioning',
+      module: './ProvisioningWizard',
+      // processor: (val) => val, // optional
+    },
+    {},
+    {}
+  );
 
-  const uploadStatus = props.imageStatus
-    ? props.imageStatus.upload_status
-    : undefined;
-  if (uploadStatus) {
-    if (uploadStatus.type === 'aws') {
-      const url =
-        'https://console.aws.amazon.com/ec2/v2/home?region=' +
-        uploadStatus.options.region +
-        '#LaunchInstanceWizard:ami=' +
-        uploadStatus.options.ami;
-      return (
-        <Button
-          component="a"
-          target="_blank"
-          variant="link"
-          icon={<ExternalLinkAltIcon />}
-          iconPosition="right"
-          isInline
-          href={url}
-        >
-          Launch instance
-        </Button>
-      );
-    } else if (uploadStatus.type === 'azure') {
-      const url =
-        'https://portal.azure.com/#@' +
-        props.uploadOptions.tenant_id +
-        '/resource/subscriptions/' +
-        props.uploadOptions.subscription_id +
-        '/resourceGroups/' +
-        props.uploadOptions.resource_group +
-        '/providers/Microsoft.Compute/images/' +
-        uploadStatus.options.image_name;
-      return (
-        <Button
-          component="a"
-          target="_blank"
-          variant="link"
-          icon={<ExternalLinkAltIcon />}
-          iconPosition="right"
-          isInline
-          href={url}
-        >
-          View uploaded image
+  if ((!error && !insights.chrome.isProd()) || insights.chrome.isBeta()) {
```

Comment:
I think this should also check if the image type/upload type is 'aws' no?

Reply:
right, good catch :+1: